### PR TITLE
[circle-opselector] Include original output

### PR DESCRIPTION
**Pull request by jinevening**
This includes original output to the generated model.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

### CODE REVIEW

File: compiler/circle-opselector/src/OpSelector.cpp [hunk 1]
```diff
@@ -169,8 +173,19 @@ std::unique_ptr<loco::Graph> make_graph(const std::vector<const luci::CircleNode
...
+      if (std::find(original_outputs.begin(), original_outputs.end(), o) != original_outputs.end())
+      {
+        originalOutput = true;
+        break;
+      }
+    }
+
     // the node isn't graph output if it is an other node's output
-    if (beingUsed)
+    if (beingUsed and not originalOutput)
```
Comment (jinevening) [inline compiler/circle-opselector/src/OpSelector.cpp:188]: Previous code does not create output if it is used by another node. This PR fixes the behavior.

File: compiler/circle-opselector/src/OpSelector.cpp [hunk 2]
```diff
@@ -156,6 +157,9 @@ std::unique_ptr<loco::Graph> make_graph(const std::vector<const luci::CircleNode
       }
     }
   }
+
+  const auto original_outputs = loco::output_nodes(g);
```
Comment (mhs4670go) [inline compiler/circle-opselector/src/OpSelector.cpp:161]: The nodes already have its graph.
```suggestion
  const auto original_outputs = loco::output_nodes(const_cast<loco::Graph *>(nodes.at(0)->graph());
```

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**